.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 npm-debug.log
+# Jetbrains IDE
+.idea/
```

Comment:
This should probably be removed, we don't want to pollute repo with opinionated files

Reply:
This does exactly what you mentioned: it prevents the repo from being polluted with opinionated files by adding them into `.gitignore`

Reply:
As no other IDE's files are listed here, adding a single one can be seen as opinionated. This addition makes sense, just not without the inclusion of other IDE's config dirs - and definately not as part of this merge request.

Developers can just as easily add the entry/entries corresponding to their editor of choice to the local .git/config file.

Reply:
I see your point, yes that makes more sense.

Reply:
Because a lot of people don't realize this and might find the info helpful:

These kinds of things can be placed in a .gitignore in a parent directory, such as (typically) your home directory. (If you're a macOS user, you probably want a .gitignore file in your home directory with an entry for .DS_Store.)